04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -319,17 +331,13 @@ RGB_stack_HARV[[1]]
 hist(RGB_stack_HARV,
      maxpixels=ncell(RGB_stack_HARV))
 
```

Comment:
it's useful to know how to just plot one band - especially when you have more than 3 bands like - 426 in our spectrometer data. Did you remove the code below for a particular reason?